Expand Difficulty Reduction mod articles + Add "Mods (lazer)" on the Main page (#12600)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Easy_(lazer)/en.md

*For the osu!stable version of this article, see: [Easy (mod)](/wiki/Gameplay/Game_modifier/Easy)*\
*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Easy** mod attempts to make the gameplay on any [beatmap](/wiki/Beatmap) played easier by halving the difficulty settings. In all [game modes](/wiki/Game_mode) except [osu!taiko](/wiki/Game_mode/osu!taiko), the mod allows you to have two extra lives in case your [health bar](/wiki/Client/Interface/Health_bar) drops to the zero, which refills it completely and instantly.

Reviewer: shouldn't use "you" on the wiki

```suggestion
The **Easy** mod attempts to make the gameplay on any [beatmap](/wiki/Beatmap) played easier by halving each difficulty setting. In all [game modes](/wiki/Game_mode) except [osu!taiko](/wiki/Game_mode/osu!taiko), the mod gives two extra lives in case the [health bar](/wiki/Client/Interface/Health_bar) drops to zero, which instead refills it instantly.
```

## Customisation

![Easy customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/EZ.jpg)

> - **Extra Lives** (0–10, default 2): Lets you choose the number of lives you want to obtain when starting a beatmap.

Reviewer: ```suggestion
- **Extra Lives** (0–10, default 2): The number of extra lives when starting a beatmap.
```
